gmplot/gmplot.py

```diff
@@ -145,6 +147,7 @@ def _process_heatmap_kwargs(self, settings_dict):
         settings_string = ''
         settings_string += "heatmap.set('threshold', %d);\n" % settings_dict['threshold']
         settings_string += "heatmap.set('radius', %d);\n" % settings_dict['radius']
+        settings_string += "heatmap.set('maxIntensity', %d);\n" % settings_dict['maxIntensity']
```

Comment:
Hi @vgm64 @ecgill,

I found this raises an error when the `maxIntensity` is omitted:
`TypeError: %d format: a number is required, not NoneType`

I think this could be fixed by prefixing this line with something like `if settings_dict['maxIntensity'] is not None:`

Code to reproduce the error:

`
import gmplot
import numpy as np

lat_center = 37.766956
lon_center = -122.438481
lat = lat_center + np.random.rand(100,1)
lon = lon_center + np.random.rand(100,1)

# Place map
gmap = gmplot.GoogleMapPlotter(lat_center, lon_center, 13)

# Heapmap
gmap.heatmap(lat, lon)

# Draw
gmap.draw("my_map.html")
`

Reply:
You're very right. #67 changes the default value of maxIntensity to 1 from None. Same basic fix but fewer lines of code.

Reply:
Thanks @vgm64 for the prompt fix!